lab-davide/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "lab-davide",
+  "version": "0.1.0",
+  "description": "",
+  "main": "server.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "mocha",
+    "start": "node server.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/FazioD/lab-07-08-single-resource-api.git"
+  },
+  "author": "faziod@uw.edu",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/FazioD/lab-07-08-single-resource-api/issues"
+  },
+  "homepage": "https://github.com/FazioD/lab-07-08-single-resource-api#readme",
+  "devDependencies": {
```

Comment:
You gotta require gulp-eslint, otherwise your gulpfile breaks immediately. I'm not sure why this didn't break travis.